added the rest of the Lob brand colors from https://www.lob.com/style…

Ryan-McBride opened the pull request:
## JIRA

https://lobsters.atlassian.net/browse/PAR-307

<!--
## Technical Design Doc
- link to TDD if the feature had one (optional)
-->

## Description

* Adds the rest of the Brand colors from the Lob styleguide https://www.lob.com/style-guide

## Screenshots
<img width="386" alt="Screen Shot 2021-05-26 at 11 07 18 AM" src="https://user-images.githubusercontent.com/3507287/119709966-ae6d0080-be12-11eb-9d1d-8b1b1e15e0bb.png">

<!--
## Tests
- info about what was used to validate the feature (more for larger, more complicated features)
-->

<!--
## Areas of Concern
- to call out what could be a problem (optional)
-->


## Questions
- Hey! Just dipping my toes in to this library right now so please let me know if I've done anything wrong


<!--
## GIFs/Memes
- (optional)
-->

## Reviewer Checklist

This section is to be filled out by reviewers

### Testing

- [ ] This code was tested by somebody other than the developer. Do not merge until this has been done.

mercedesb commented:
Most of these colors are already in this file. We changed the names and added modifiers to follow more standard Tailwind conventions to make it easier for devs to ramp up. For example, if you search for `#0099d7` (cerulean), you'll see that it's the default blue, etc.

Ryan-McBride commented:
> Most of these colors are already in this file. We changed the names and added modifiers to follow more standard Tailwind conventions to make it easier for devs to ramp up. For example, if you search for `#0099d7` (cerulean), you'll see that it's the default blue, etc.

alright, I'll remove the duplicates

Ryan-McBride commented:
duplicates have been removed

mercedesb commented:
Maybe we should talk and see if these can be incorporated into the existing "gradients" so that future devs won't need to remember the funky names and can stick to the simple name + modifier extension

bethqiang commented:
+1 to Mercedes' comment - I personally don't want to have to sit and wonder, "Is it azure or bluebird or that other weird name that I can't remember off the top of my head?"

Other tangential comment/q - in the link for [the design system colors tab](https://www.figma.com/file/PFyulPE8zYVuRxRAhnhd7T/LOB-Design-System-(Updated)?node-id=0%3A1) and the style guide, it says that these are `Tertiary Colors` and `Only use these colors for spot illustrations and misc. graphics` - have we confirmed with ...someone that we're going to be using these throughout our platforms? For example - these extra colors don't show up anywhere in the [design systems UI components Figma file](https://www.figma.com/file/PFyulPE8zYVuRxRAhnhd7T/LOB-Design-System-(Updated)?node-id=39%3A0).

Ryan-McBride commented:
Potentially? All my ticket (PAR-304) say is to import the colors, doesn't specify any to ignore. Additionally I feel like it would be easier to match colors if they have the names listed in the Lob branding wouldn't it?

Ryan-McBride commented:
I'll just stick with the existing colors for now
